packages/ember-glimmer/lib/helpers/inline-if.js

```diff
@@ -0,0 +1,11 @@
+import { toBool as emberToBool } from './if-unless';
+
+//TODO: GJ: docs
```

Comment:
And thanks for noting this. In 1.13 we lost a ton of docs b/c PRs were merged without this stuff. Definitely worth doing up front.